opendata_module/opmon_opendata/api/urls.py

```diff
@@ -24,9 +24,10 @@
 from . import views
 
 urlpatterns = [
-    path('heartbeat/', views.heartbeat),
-    path('daily_logs/', views.get_daily_logs),
-    path('logs_sample/', views.get_preview_data),
-    path('date_range/', views.get_date_range),
-    path('column_data/', views.get_column_data)
+    path('heartbeat', views.heartbeat),
+    path('daily_logs', views.get_daily_logs),
+    path('daily_logs_meta', views.get_daily_logs_meta),
+    path('logs_sample', views.get_preview_data),
+    path('date_range', views.get_date_range),
+    path('column_data', views.get_column_data)
```

Comment:
Do we really want to get rid of the ending slash?

Reply:
As noted in the commit message "paths cause redirects and curl examples in docs do not work". And code was updated to match API documentation.